catalog/resource_volume.go

```diff
@@ -0,0 +1,100 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// This structure contains the fields of catalog.UpdateVolumeRequestContent and catalog.CreateVolumeRequestContent
+// We need to create this because we need Owner, FullNameArg, SchemaName and CatalogName which aren't present in a single of them.
+// We also need to annotate tf:"computed" for the Owner field.
+type VolumeInfo struct {
+	// The name of the catalog where the schema and the volume are
+	CatalogName string `json:"catalog_name"`
```

Comment:
please run `make fmt`

Reply:
This is not required, it's no-op and fmt test was passed https://github.com/databricks/terraform-provider-databricks/actions/runs/5131170653/jobs/9230902643